p2p/node.go

```diff
@@ -193,6 +196,9 @@ func New(ctx context.Context, config Config) (*Node, error) {
 	if config.PerPeerPubSubMessageBurst == 0 {
 		config.PerPeerPubSubMessageBurst = defaultPerPeerPubSubMessageBurst
 	}
+	if config.MaxBytesPerSecond == 0 {
+		config.MaxBytesPerSecond = defaultMaxBytesPerSecond
```

Comment:
@albrow Should we use a different default, or do you think that 1 MB still makes sense?

Reply:
Let's go ahead and bump it to 5MB.